tests/scalability/scale-test.sh

```diff
@@ -2,21 +2,21 @@
 
 source ./util.sh
 
-GSID=85ffe8da-c82f-4035-86c5-9d2b5f42d6f7
+GSB_NAME=gameserverbuild-sample-openarena
 
 echo "test 1: scale up to 16 servers from 1 standby server"
 kubectl apply -f ./standby/1.yaml
```

Comment:
these kubectl apply commands are not valid anymore, right?

Reply:
It creates the gsb with 1 initial standby and 16 max. I think we still need it.

Reply:
oh, sorry, files were not visible and I thought they were deleted. Do we need all three of them now?

Reply:
I think we need, but it shouldn't be different initial standby. We actually need to test the scale time for different max.

Reply:
max in all three files is currently the same, 16. Plus, we can send a JSON patch via kubectl and change it dynamically if needed. Not a big deal TBH, I wouldn't block this PR for it.

Reply:
Updated test cases to use max as the variant.